wrapper/Makefile

```diff
@@ -31,11 +33,37 @@ OBJS = \
 	$(OBJDIR)/darray.o \
     $(OBJDIR)/main.o \
 	$(OBJDIR)/pugixml.o \
+	
+LIBOBJS = \
+	$(LIBOBJDIR)/alloc.o \
+	$(LIBOBJDIR)/file_io.o \
+	$(LIBOBJDIR)/cpu.o \
+	$(LIBOBJDIR)/convolution.o \
+	$(LIBOBJDIR)/adm.o \
+	$(LIBOBJDIR)/adm_tools.o \
+	$(LIBOBJDIR)/ansnr.o \
+	$(LIBOBJDIR)/ansnr_tools.o \
+	$(LIBOBJDIR)/vif.o \
+	$(LIBOBJDIR)/vif_tools.o \
+	$(LIBOBJDIR)/motion.o \
+	$(LIBOBJDIR)/psnr.o \
+	$(LIBOBJDIR)/math_utils.o \
+	$(LIBOBJDIR)/convolve.o \
+	$(LIBOBJDIR)/decimate.o \
+	$(LIBOBJDIR)/ssim_tools.o \
+	$(LIBOBJDIR)/ssim.o \
+	$(LIBOBJDIR)/ms_ssim.o \
+    $(LIBOBJDIR)/svm.o \
+	$(LIBOBJDIR)/darray.o \
+	$(LIBOBJDIR)/pugixml.o \
+	$(LIBOBJDIR)/combo_lib.o \
+	$(LIBOBJDIR)/vmaf_lib.o \
+    $(LIBOBJDIR)/libvmaf.o \
 
 AVX_OBJS := \
 	$(OBJDIR)/convolution_avx.o \
 
-all: vmafossexec
+all: vmafossexec lib
 
```

Comment:
You don't have to include lib in all. It can be called from the top-level Makefile by 'make lib'.